app/questionnaire/router.py

```diff
@@ -246,6 +221,11 @@ def full_routing_path(self):
                 )
         return full_routing_path
 
+    def can_access_section_summary(self, section_id, list_item_id):
```

Comment:
I wonder if is_location_valid needed with can_access_section_summary already there. When the section end point gets hit, this will determine whether the survey is complete and has a summary, if it returns false then the get_section function in questionnaire will redirect it to the correct block. 

My issue would be if we did have a is_location_valid like the other blocks which raises an exception, that if someone saved and resumed and there was on the hub, that the click to sections/<section_id>/ which would instantiate a SectionSummary handler and raise an exception most times if we followed the current pattern.